array_absl.h

```diff
@@ -0,0 +1,14 @@
+#include "absl/strings/str_format.h"
+#include "array.h"
+
+// Adds Abseil Stringify support (https://abseil.io/blog/20221115-stringify).
+
+namespace nda {
+
+// interval -> string.
+template <typename Sink, index_t Min_ = dynamic, index_t Extent_ = dynamic>
+void AbslStringify(Sink& sink, const nda::interval<Min_, Extent_>& p) {
+  absl::Format(&sink, "[%v, %v] (extent = %v)", p.min(), p.max(), p.extent());
```

Comment:
I'm not sure about including the `(extent = )` part, I think if this were used with `absl::StrJoin(vector<nda::interval<...>>)`, it would produce a lot of noise?

Reply:
I agree. I mostly wanted to get your opinion on this. The closed interval is easy to implement with `max()`, but I'm really tempted to use the open interval. So then a lot of the time, the extent is implicit and you can do subtraction, etc.

E.g., `min = 0, extent = 256` would display at `[0, 256)` so you know it's an image of width 256 and you can subtract.

WDYT?

Reply:
I think half open is good too, it also matches the typical iterator/loop conventions in c++